src/Admin/BreadcrumbsBuilder.php

```diff
@@ -46,7 +46,7 @@ public function configureOptions(OptionsResolver $resolver): void
         ]);
     }
 
-    public function getBreadcrumbs(AdminInterface $admin, string $action): iterable
+    public function getBreadcrumbs(AdminInterface $admin, string $action): array
```

Comment:
this one does not seem right

Reply:
See https://github.com/sonata-project/SonataAdminBundle/pull/6379/files#diff-d034497868502fe990e3db10a7c2eefbR33

Type was changed in 3.x from iterable to array.
So, return typehint has to follow.

If you want to rollback this change, it should start with 3.x and not block this merge.

Reply:
Thats the problem changing phpdoc without real testing those types, lets hope we dont have to deal with a lot more prs on 3.x about that